tests/integration/aof.tcl

```diff
@@ -368,4 +368,52 @@ tags {"aof external:skip"} {
             assert_equal 2 [$client llen mylist3]
         }
     }
+
+    # Test that ZMPOP/BZMPOP work fine with AOF.
+    create_aof {
+        append_to_aof [formatCommand zadd myzset 1 one 2 two 3 three]
+        append_to_aof [formatCommand zadd myzset2 4 four 5 five 6 six]
+        append_to_aof [formatCommand zadd myzset3 1 one 2 two 3 three 4 four 5 five]
+    }
+
+    start_server_aof [list dir $server_path aof-load-truncated no] {
+        test "AOF+ZMPOP/BZMPOP: pop elements from the zset" {
+            set client [redis [dict get $srv host] [dict get $srv port] 0 $::tls]
+            set client2 [redis [dict get $srv host] [dict get $srv port] 1 $::tls]
+            wait_done_loading $client
+            wait_done_loading $client2
```

Comment:
why do we need to wait for loading to complete twice?
let's remove the excessive line from the BLMPOP test as well.,